common/Storage.py

```diff
@@ -168,7 +168,7 @@ def _load_tensors(self, graph):
 
             # Number of edges to be processed per batch
             num_attributes = len(vars(graph.edges[0])) + 4
-            num_edges_per_batch = 200000 // num_attributes
+            num_edges_per_batch = 20000 // num_attributes
```

Comment:
convert this constant to a constat literal?

Reply:
Added a class attribute : _MAX_MUTATIONS.